src/UIView/UIView+NibInitable.m

```diff
@@ -0,0 +1,28 @@
+//
+//  UIView+NibInitable.m
+//  Chronability
+//
+//  Created by Logan Wright on 4/29/15.
+//  Copyright (c) 2015 IntrepidPursuits. All rights reserved.
+//
+
+#import "UIView+NibInitable.h"
+
+@implementation UIView (NibInitable)
+- (instancetype)initWithNibNamed:(NSString *)nibNameOrNil
+{
+    if (!nibNameOrNil) {
+        nibNameOrNil = NSStringFromClass([self class]);
+    }
+    NSArray *viewsInNib = [[NSBundle mainBundle] loadNibNamed:nibNameOrNil
+                                                        owner:self
+                                                      options:nil];
+    for (id view in viewsInNib) {
+        if ([view isKindOfClass:[self class]]) {
+            self = view;
+            break;
+        }
+    }
+    return self;
```

Comment:
I feel like it might be worth to assert here...what happens when the view is not found?